src/fragments/lib/auth/native_common/signin_with_custom_flow/common.mdx

```diff
@@ -1,49 +1,71 @@
-The Auth category can be configured to perform a custom authentication flow defined by you. The following guide shows how to setup a simple passwordless authentication flow.
+The Auth category can be configured to perform a [custom authentication flow](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-challenge.html) defined by you. The following guide shows how to setup a simple passwordless authentication flow.
 
 ## Prerequisites
 
-import ios0 from "/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx";
+import ios0 from '/src/fragments/lib/auth/ios/getting_started/10_preReq.mdx';
 
-<Fragments fragments={{ios: ios0}} />
+<Fragments fragments={{ ios: ios0 }} />
+
+import flutter0 from '/src/fragments/lib/auth/flutter/getting_started/10_preReq.mdx';
+
+<Fragments fragments={{ flutter: flutter0 }} />
 
 ## Configure Auth Category
 
-import ios1 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx";
+import ios1 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ ios: ios1 }} />
 
-<Fragments fragments={{ios: ios1}} />
+import flutter1 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/10_cli_setup.mdx';
+
+<Fragments fragments={{ flutter: flutter1 }} />
 
 ## Register a user
 
 The CLI flow as mentioned above requires a username and a valid email id as parameters to register a user. Invoke the following api to initiate a sign up flow.
 
-import ios2 from "/src/fragments/lib/auth/ios/signin_with_custom_flow/20_signup.mdx";
+import ios2 from '/src/fragments/lib/auth/ios/signin_with_custom_flow/20_signup.mdx';
+
+<Fragments fragments={{ ios: ios2 }} />
+
+import flutter2 from '/src/fragments/lib/auth/flutter/signin_with_custom_flow/20_signup.mdx';
 
-<Fragments fragments={{ios: ios2}} />
+<Fragments fragments={{ flutter: flutter2 }} />
 
 The next step in the sign up flow is to confirm the user. A confirmation code will be sent to the email id provided during sign up. Enter the confirmation code received via email in the `confirmSignUp` call.
```

Comment:
What happens if a user leaves the experience before confirming the user? if they try to login again, would they preasented with a challenge to verify their user?

Reply:
The library throws a UserNotConfirmedException, as it does in other cases. I've added an integration test to confirm this.